src/js/components/jobs/detail.tsx

```diff
@@ -332,16 +332,14 @@ class JobDetail extends React.Component<Props, State> {
                 />
               }
               postMessage={<JobMessage job={job} />}
-              backLink={
-                job.status === CONSTANTS.STATUS.STARTED ? null : (
-                  <BackLink
-                    label={i18n.t('Install another product')}
-                    url={routes.product_list()}
-                    className="slds-p-top_small"
-                  />
-                )
-              }
             />
+            {job.status === CONSTANTS.STATUS.STARTED ? null : (
+              <BackLink
+                label={i18n.t('Install another product')}
+                url={routes.product_list()}
+                className="slds-p-top_small"
+              />
+            )}
```

Comment:
@jgerigmeyer this change is because the new button to spin an org is visually within the `<Intro />` component between the `backlink` and `cta`.  I felt it made sense to separate them to accommodate the layout.

Reply:
Ok, I see that. I would consider that this new button should be _part_ of the cta component(s)?